examples/usecases/multi-gpu/hvd_wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Could we have our own version of `horovodrun` that includes this code? That would simplify our examples as well

Reply:
I agree, that wold be great, but I couldn't a way to do this. Setting of these environment variables has to come after `horovodrun` and seems to depend heavily on the order of imports so it has come before everything else (likely tensorflow). I wasn't able to find a solution in an afternoon, and I'm not sure if it's worth digging more. I think we will have to pretty much solve the import order problem, which seems difficult to analyze and debug.

Reply:
Makes sense. In that case we can leave that task for something to work on after this PR.